### PR TITLE
[AAP-21949] Login page

### PR DESCRIPTION
**Pull request by ShaiahWren**
This pr creates a newly styled login page, adding product logos and an ansible mark. 
Screenshot 2024-03-27 at 4 24 29 PM
Screenshot 2024-03-27 at 3 59 52 PM
Screenshot 2024-03-27 at 4 18 33 PM

### CODE REVIEW

File: frontend/common/Login.tsx [hunk 1]
```diff
@@ -45,17 +68,32 @@ export function Login(props: LoginProps) {
...
+              authOptions={props.authOptions}
+              onLoginUrl={props.onLoginUrl}
+              hideInputs={props.hideInputs}
+            />
+          </Inner>
+        </LeftHalf>
+        <RightHalf>
+          <ServiceLogo
+            src={
+              process.env.PRODUCT === 'Automation Hub'
```
Comment (nixocio) [inline frontend/common/Login.tsx:87]: What about we create a data structure to hold those strings, either an object or an enum?

Something like 


```tsx
const projectNames = {
  ansible: 'Ansible',
  automationHub: 'Automation Hub',
  eventDrivenAutomation: 'Event Driven Automation',
  awx: 'AWX',
};
```

Comment (aratti96) [reply frontend/common/Login.tsx:87]: We don't want to use that background image anymore. The background should be a solid color.
dark mode: #151515
light mode: #F0F0F0

Comment (ShaiahWren) [reply frontend/common/Login.tsx:87]: hmm.. I think that the data coming from "process.env.PRODUCT" is already human readable- it's the translation to the logo.svg that I can translate.

**Review by aratti96 — APPROVED** [review]

**Comment by nixocio** [comment]
This one was closed in favor of https://github.com/ansible/ansible-ui/pull/2133

**Pull request closed without merging**